### PR TITLE
Fix: Align security styling with design

### PR DESCRIPTION
**Pull request by Mikehrn**
Aligns the security tab with the designs and fixes some minor bugs, we can probably refactor this a bit further (more logic can move to a composable and beter fragmenting), but we can address after Sep 2

Screenshot 2024-08-27 at 23 35 42

### CODE REVIEW

File: packages/frontend-2/components/settings/workspaces/Security.vue [hunk 1]
```diff
@@ -6,51 +6,90 @@
...
-                {{ `@${item.domain}` }}
-              </span>
-            </template>
-            <template #delete="{ item }">
-              <FormButton color="danger" @click="() => openRemoveDialog(item)">
-                Delete
-              </FormButton>
-            </template>
-          </LayoutTable>
+            <p class="font-body-xs font-medium flex-1">@{{ domain.domain }}</p>
```
Comment (andrewwallacespeckle) [inline packages/frontend-2/components/settings/workspaces/Security.vue:16]: text-body-xs

**Review by andrewwallacespeckle — CHANGES_REQUESTED** [review]
Nice work! 1 small change then this is good to go

**Review by andrewwallacespeckle — APPROVED** [review]

**Review by cdriesler — APPROVED** [review]
image

**Comment by Mikehrn** [comment]
> image

Screenshot 2024-08-28 at 13 25 38

**Pull request merged**